Integration/ClientIntegration.php

```diff
@@ -800,12 +801,31 @@ private function handleException(\Exception $exception)
             if ($errorData) {
                 $this->setLogs($errorData, $exception->getStatType());
             }
+        } elseif ($exception instanceof ConnectException) {
+            if (function_exists('newrelic_notice_error')) {
+                call_user_func(
+                    'newrelic_notice_error',
+                    'ContactClient Connection Error: '.$this->contactClient->getName().' ['.$this->contactClient->getId(
```

Comment:
Double quotes would make this easier to read imo 
`"ContactClient Connection Error: {$this->contactClient->getName()} [{$this->contactClient->getId()}] in Campaign: {$this->campaign->getName()} [{$this->campaign->getId()}]",`

Reply:
I feel like this is debatable. Either could be more readible depending on your IDE and personal training. There's no code-style enforcing either convention so I wouldn't reject a PR for it.